### PR TITLE
Overview page health requests merged into one per cluster

### PR DESCRIPTION
**Pull request by hhovsepy**
### Describe the change
Overview page was sending a 'health' request per namespace, now it sends per cluster for a list of given namespaces.

### Steps to test the PR
Similar to other optimization PRs testing

Creating ~200 namespaces with something like:

`for i in {1..200}; do kubectl create ns test-$i; done`

Visit the Overview page.
Compare previous time it takes to load all the requests and the page with this PR.

Cleanup with:

`for i in {1..200}; do kubectl delete ns test-$i --wait=false --ignore-not-found=true; done`

### Automation testing

Appropriate **unit / integration / cypress** tests are modified or added new ones.

### Issue reference

https://github.com/kiali/kiali/issues/7076

### CODE REVIEW

**Review by josunect — COMMENTED** [review]
In the current Kiali, there are meny health requests, page takes 11.85s loading, 234 total requests: 

image

With current PR, the amount of requests is reduced to 33, and 8.32s load: 

image

I see the validations and health are the requests taking longer: 
image

File: frontend/src/pages/Mesh/target/TargetPanelNamespace.tsx [hunk 1]
```diff
@@ -788,8 +788,9 @@ export class TargetPanelNamespace extends React.Component<TargetPanelNamespacePr
   }
 
   private fetchHealthStatus(): Promise<void> {
-    return API.getNamespaceAppHealth(this.namespace, this.props.duration, this.cluster)
-      .then(rs => {
+    return API.getClustersAppHealth(this.namespace, this.props.duration, this.cluster)
+      .then(results => {
+        console.log('In HealthStatus');
```
Comment (josunect) [inline frontend/src/pages/Mesh/target/TargetPanelNamespace.tsx:793]: This is needed?

Comment (hhovsepy) [reply frontend/src/pages/Mesh/target/TargetPanelNamespace.tsx:793]: odd from debug

**Review by ferhoyos — APPROVED** [review]
The code looks good and the performance improvement is clear.

**Pull request merged**